### DISCUSSION TITLE
My student developer pack application keeps getting rejected

### DISCUSSION THREAD

**Discussion by Queln** [community/community#66061] [GitHub Education] [answered]
I have applied for the developer student pack a few times now, however, I've finally gotten everything I need to apply. But I keep being told that I am "unlikely to be verified until I've completed my Github profile with my full name exactly as it appears in my academic affiliation document". But I already have done so I believe.

Please let me know if I am doing something wrong or it's an issue out of my control.

**Comment by Gatix1** [comment]
Is the name on GitHub and student card the same? Is it exactly the same as on the student card?

**Reply by Queln** [reply]
Yes it is the exact same. Even the order of my first and last name.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @FilipBravenec , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Reply by Queln** [reply]
Thank you, I didn't know which one to give it. Sorry.

**Comment by Queln** [comment]
I think it's because of github being unable to use my back camera meaning the name is unreadable least from what I can tell, but what am I supposed to do about it?

**Reply by Queln** [reply, marked as answer]
I was correct. I took a photo using the back camera and uploaded it to my computer so I can use it instead and I was approved within a few moments.
